.gitignore

```
@@ -4,6 +4,9 @@ target/
 !**/src/main/**/target/
 !**/src/test/**/target/
 
+# Ignore environment files
+.env.template
```

```suggestion
.env
```
ignore .env and push .env.template with variables without values

---

Thanks! It just changed after I refactored .env